playbooks/roles/mattermail/templates/mattermail.service

```diff
@@ -0,0 +1,17 @@
+# mattermail
+[Unit]
+Description=mattermail server
+After=network-online.target
+
+[Service]
+Type=simple
+WorkingDirectory={{ MATTERMAIL_HOME }}
```

Comment:
Nit: is this needed if we're already specifying `{{ MATTERMAIL_HOME }}` before `/bin/mattermail` right below?

Reply:
I see you got this from the README on their repo. Still, I wonder if this is necessary.

Reply:
Yep, I agree. Probably unneeded, I'll check if any issues arises without it.